src/main.ts

```diff
@@ -1,8 +1,19 @@
 import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app.module';
+import { DocumentBuilder, SwaggerModule } from '@nestjs/swagger';
 
 async function bootstrap() {
     const app = await NestFactory.create(AppModule);
+
+    const config = new DocumentBuilder()
+    .setTitle('Cats example')
```

Comment:
@PCalape The indentation looks off, was this file formatted by Prettier?

Reply:
Hello @arnellebalane! Sorry i forgot to setup prettier formatting upon file saving in my ide. Applied the formatting for all thanks!